bin/prettier.js

```diff
@@ -36,13 +36,15 @@ const argv = minimist(process.argv.slice(2), {
   alias: { help: "h", version: "v", "list-different": "l" },
   unknown: param => {
     if (param.startsWith("-")) {
+      // eslint-disable-next-line no-console
```

Comment:
Another way would be to add `/* eslint-disable no-console */` at the top of bin/prettier.js and scripts/sync-flow-tests.js.